Detectors/TRD/trap/test_trap_tpp.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "trap_tpp.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+// $Id$:
+
+int main(int argc, char** argv)
```

Comment:
what is this doing? It does not seem to be a unit test ... so maybe call it differently than "test_".

Reply:
these are files for Venelin to have a stand alone c++ program for implementation of the assembler. I needed somewhere to put them so figured this is as good as any other.
A readme in the directory is probably called for. Else one up to explain the directory structure.

Reply:
Second thoughts will remove and put in a seperate PR